File: Programas em Java/03 - Laço While e For/Script2.java
/*
 * Faca um algoritmo que conte de 1 a 100 e a cada 
 * multiplo de 10 emita uma mensagem: "Multiplo de 10".
 */

package geek;

public class Script2 {

	public static void main(String[] args) {
		for(int i = 1; i <= 100; i++) {
			System.out.println(i);
			if(i % 10 == 0) {
				System.out.println("Múltiplo de 10");
			}
		}
	}

}

File: Programas em Python/03 - Laço While e For/Script2.py
# Faca um algoritmo que conte de 1 a 100 e a cada 
# multiplo de 10 emita uma mensagem: "Multiplo de 
# 10".

#processamento
for n in range(1, 101): #n - 1
    print(n)
    if n % 10 == 0:
        print("Múltiplo de 10")
